.github/workflows/release_build.yml

```diff
@@ -27,4 +27,33 @@ jobs:
       - name: Github Release
         uses: softprops/action-gh-release@v1
         with:
-          files: "./build/*.zip"
\ No newline at end of file
+          files: "./build/*.zip"
+  build-Docker:
```

Comment:
Create workflow which would be able to build the image from any branch using workflow_dispatch event, so we can test changes from any branch.

Reply:
added repo dispatch, but tag could be pushed anywhere